amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/reports/designer/ReportColumnProvider.java

```diff
@@ -139,7 +139,7 @@ public class ReportColumnProvider extends ReportEntityProvider {
     private static final Comparator<AmpColumns> ME_COLS_COMPARATOR =
             Comparator.comparing(c -> ME_COLUMNS_ORDER.getOrDefault(c.getColumnName(), DEFAULT_ORDER_VALUE));
 
-    private TranslatorService translatorService;
+    public TranslatorService translatorService;
```

Comment:
to keep the same structure and since translator service is a singleton instead of making it punlaick create a variable inside ReportColumValidator and populate it at creation time

Reply:
done